app/main/views/briefs.py

```diff
@@ -347,7 +347,12 @@ def view_response_result(brief_id):
 
     if len(brief_response) == 0:
         return redirect(url_for(".brief_response", brief_id=brief_id))
-    elif brief_response[0].get('essentialRequirementsMet') or all(brief_response[0]['essentialRequirements']):
+
+    # The legacy brief response data format has a booleans for each essential requirement
+    # rather than a single flag
+    if ('essentialRequirementsMet' not in brief_response[0]
+        and all(brief_response[0]['essentialRequirements'])
+            or brief_response[0].get('essentialRequirementsMet')):
```

Comment:
That's the version I replaced(!) Turns out my replacing it was wrong. I did so to deal with a situation where you get to this page without `essentialRequirementsMet` being `False` which, it turns out isn't possible with the new flow.

I'll revert this to the older version which should solve it. Good spot.